RunningOrder/Sprint/SprintList.swift

```diff
@@ -13,31 +13,45 @@ extension Sprint: Identifiable {
 }
 
 struct SprintList: View {
-    @State private var sprints: [Sprint] = []
+    @State private var sprints: [Sprint] = [Sprint.Previews.sprints[0]]
     @State private var showNewSprintModal = false
 
     var body: some View {
-        List {
-            Section(header: Text("Active Sprints")) {
-                ForEach(sprints) { sprint in
-                    NavigationLink(
-                        destination: StoryList(stories: sprint.stories)
-                            .listStyle(PlainListStyle()),
-                        label: {
-                            Text(sprint.name)
-                        })
+        VStack(alignment: .leading) {
+            List {
+                Section(header: Text("Active Sprints")) {
+                    ForEach(sprints) { sprint in
+                        NavigationLink(
+                            destination: StoryList(stories: sprint.stories)
+                                .listStyle(PlainListStyle()),
+                            label: {
+                                HStack {
+                                    SprintNumber(number: sprint.number, color: sprint.color)
+                                    Text(sprint.name)
+                                }
+                            })
+                    }
                 }
+                Section(header: Text("Old Sprints")) {
+                    EmptyView()
+                }
+            }.sheet(isPresented: $showNewSprintModal) {
+                NewSprintView(createdSprint: self.$sprints.appendedElement)
             }
 
             Button(action: { self.showNewSprintModal.toggle() }) {
-                Text("New Sprint")
-            }
-            Section(header: Text("Old Sprints")) {
-                EmptyView()
+                HStack {
+                    Image(nsImage: NSImage(named: NSImage.addTemplateName)!)
```

Comment:
use Image with named directly no ?

Reply:
maybe with resizable() modifier

Reply:
`Image(NSImage.addTemplateName)`is not working since there are not SystemImage / SFSymbols in this version

Reply:
Why Resizable on the image ? The cross would fill all the circle

Reply:
aaaah NSImage.addTemplateName is a generic existing image !! :O ok, I tested `Image(NSImage.addTemplateName)` it doesn't work...